config/action.d/firewallcmd-allports.conf

```diff
@@ -0,0 +1,52 @@
+# Fail2Ban configuration file
+#
+# Author: Donald Yandt 
+# Because of the --remove-rules in stop this action requires firewalld-0.3.8+
+
+
+[INCLUDES]
+
+before = iptables-blocktype.conf
+
+actionstart = firewall-cmd --direct --add-chain ipv4 filter f2b-<name>
+              firewall-cmd --direct --add-rule ipv4 filter f2b-<name> 1000 -j RETURN
+              firewall-cmd --direct --add-rule ipv4 filter <chain> 0 -j f2b-<name>
+
+actionstop = firewall-cmd --direct --remove-rule ipv4 filter <chain> 0 -j f2b-<name>
+             firewall-cmd --direct --remove-rules ipv4 filter f2b-<name>
+             firewall-cmd --direct --remove-chain ipv4 filter f2b-<name>
+
+
+# Note:  uses regular expression word boundaries '\b' 
+# Example actioncheck: firewall-cmd --direct --get-chains ipv4 filter | grep -q '\bf2b-recidive\b'
+
+actioncheck = firewall-cmd --direct --get-chains ipv4 filter | grep -q '\bf2b-<name>\b'
```

Comment:
I guess ```\b``` is not quite proper here:
```
> /bin/ls | grep '\bfail2ban-regex\b' 
fail2ban-regex-0.9
fail2ban-regex-oldmaster
```
if it is at the end of the line in the output of firewall-cmd, previously used '$' was a better choice.  So why not just to make it ```\sf2b-<name>$```?